examples/src/providers/rule_closures/conditions.js

```diff
@@ -30,6 +30,32 @@ module.exports = [
       }
     },
   },
+  {
+    /**
+     * invalidOrderUdp
+     * example {closure: "invalidOrderUdp"} - If the order even valid
+     * @param type
+     * @return boolean - true if the order data is not valid JSON
+     **/
+    name: 'invalidOrderUdp',
+    handler(facts, context) {
+      try {
+        // The UDP Input gives us a special object and the property that
+        // holds our request (httpRequest).
+        // httpRequest conforms to IProviderReq and contains:
```

Comment:
```suggestion
        // udpRequest conforms to IProviderReq and contains:
```

Reply:
I reworked this section to be more like the http example and fixed the comments in the process